src/main/java/team/gif/gearscout/schedule/ScheduleController.java

```diff
@@ -0,0 +1,60 @@
+package team.gif.gearscout.schedule;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+import org.springframework.web.bind.annotation.PathVariable;
+import team.gif.gearscout.shared.UserRoles;
+import team.gif.gearscout.token.TokenService;
+import team.gif.gearscout.tba.TbaService;
+import team.gif.gearscout.tba.MatchScheduleEntry;
+
+import java.util.*;
```

Comment:
Specify each import. Though this specific case won't lead to namespace collisions, I try to avoid wildcard imports